pkg/monitoring/config.go

```diff
@@ -30,13 +30,15 @@ type SolanaConfig struct {
 }
 
 type KafkaConfig struct {
-	Brokers          string `json:"brokers,omitempty"`
-	ClientID         string `json:"client_id,omitempty"`
-	SecurityProtocol string `json:"security_protocol,omitempty"`
-	SaslMechanism    string `json:"sasl_mechanism,omitempty"`
-	SaslUsername     string `json:"sasl_username,omitempty"`
-	SaslPassword     string `json:"sasl_password,omitempty"`
-	Topic            string `json:"topic,omitempty"`
+	Brokers                  string `json:"brokers,omitempty"`
+	ClientID                 string `json:"client_id,omitempty"`
+	SecurityProtocol         string `json:"security_protocol,omitempty"`
+	SaslMechanism            string `json:"sasl_mechanism,omitempty"`
+	SaslUsername             string `json:"sasl_username,omitempty"`
+	SaslPassword             string `json:"sasl_password,omitempty"`
+	TransmissionTopic        string `json:"kafka_transmission_topic,omitempty"`
+	ConfigSetTopic           string `json:"kafka_config_set_topic,omitempty"`
+	ConfigSetSimplifiedTopic string `json:"kafka_config_set_simplified_topic,omitempty"`
```

Comment:
you don't need the `kafka_` prefix here. These field will be in an object under the `kafka` key anyway! See https://github.com/smartcontractkit/chainlink-solana/blob/feature/relay/pkg/monitoring/config.go#L17